src/test_c5t_popen2.cc

```diff
@@ -10,34 +10,34 @@
 
 TEST(Popen2Test, Smoke) {
   std::string result;
-  C5T_POPEN2({"/usr/bin/bash", "-c", "echo PASS"}, [&result](std::string const& line) { result = line; });
+  C5T_POPEN2({"bash", "-c", "echo PASS"}, [&result](std::string const& line) { result = line; });
```

Comment:
And the last change was that `/usr/bin/bash` (which was incorrectly hardcoded) is not available on OS X. I've changed it into `/bin/bash` first, but then realized that just `bash` would do fine.